src/lib/y2storage/fake_device_factory.rb

```diff
@@ -250,6 +260,53 @@ def create_disk(_parent, args)
       new_partitionable(Disk, args).name
     end
 
+    # Factory method to create a Software RAID (Md)
+    #
+    # @param _parent [nil] (Mds are toplevel)
+    # @param args [Hash] RAID parameters:
+    #   * :name [String] RAID name (e.g., /dev/md0)
+    #   * :md_level [Storage::MdLevel]
+    #   * :chunk_size [DiskSize]
+    #
+    # @return [String] name of the new Software RAID (e.g., /dev/md0)
+    def create_md(_parent, args)
+      name = args["name"]
+      name ||= Md.find_free_numeric_name(devicegraph)
+
+      levels = {
+        "raid0"  => Storage::MdLevel_RAID0,
+        "raid1"  => Storage::MdLevel_RAID1,
+        "raid4"  => Storage::MdLevel_RAID4,
+        "raid5"  => Storage::MdLevel_RAID5,
+        "raid6"  => Storage::MdLevel_RAID6,
+        "raid10" => Storage::MdLevel_RAID10
```

Comment:
why defining it here again? Why not simple to_s or to_sym ? I really do not like it as it e.g. miss container type and in general if we add another raid level it will miss here.

Reply:
Yes, you are right. Basically, I was avoiding to use wrapper methods, because all this code was written before having the wrapper. But for this specific check it makes sense to start to use it.